### DISCUSSION TITLE
How see current time position as frame number?

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#411] [Q&A] [answered]
I went ahead and switched it to mplayer (instead of mpv) and now the option in statusbar for fame counter is no longer greyed out. In my case I want to split a file at 3:32:00 but need to see what frame it's on. Where can I see that? Thanks

**Comment by smplayer-dev** [comment, marked as answer]
I'm afraid this is not supported with mplayer.

**Reply by tack32** [reply]
Really? Ok. Thanks for the reply.
